### DISCUSSION TITLE
Codespace failed to start

### DISCUSSION THREAD

**Discussion by gilds** [community/community#51690] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hello everyone. I'm trying to enter my codespace for CS50 and after clicking "Login with GitHub" I get the error: "Failed to start Codespace. Back to [code50/50615861]

I've been trying for over an hour and I don't know what to do... CS50 status says everything is operational.
Someone knows what's happening and how to solve this?

Thank you in advance.

**Comment by kbuke** [comment]
i am having this issue too. desperate for a solution

**Reply by foych587** [reply]
I am also having this same issue. I look forward to being able to work on the problems during the weekend and it's really frustrating to not have the ability to do that.

**Reply by gilds** [reply]
Tell me about it... The weekend is when I have more time to go through the course and it's impossible to do it today... I also tried to connect VS Code Desktop and sync evertyhing but the problem is with the Codespaces. I hope they will be able to fix this today.

**Reply by foych587** [reply]
Attaching a screenshot of the issue
![Screenshot 2023-04-01 122946](https://user-images.githubusercontent.com/21027544/229302982-73034dc6-bfb1-46e5-b28b-0b040d9f5922.png)

**Reply by foych587** [reply]
Same here. I tried that too. It must be an issue with a recent update. Super frustrating

**Reply by foych587** [reply]
@gilds  @kbuke  I just went to github.com/codespaces and selected "New Codespace" in the upper right and then pick the repository that begins with code50 and then that new codespace allows you to sign in. I have now been able to start working on the problem sets again. Hope that is helpful for you

![image](https://user-images.githubusercontent.com/21027544/229305014-2ad7a5e0-3baa-4141-98bb-68c3b0eef1d4.png)

![image](https://user-images.githubusercontent.com/21027544/229305054-09376c18-0585-4411-be18-a9d32372209e.png)

**Reply by gilds** [reply]
Hello @foych587! Thank you for your reply! I tried and it worked! Thank you so much! Have a nice weekend. :)

**Reply by gilds** [reply]
@foych587 Please write your answer in a new thread so I can mark it as an answer.

**Comment by foych587** [comment, marked as answer]
Hi @gilds glad that worked for you. Yes I have been able to upload all my exercises as before and submit. Starting this new thread so you can mark the comments above as an answer.

**Reply by kbuke** [reply]
hey while i can access my work now, the load times are super long, and sometimes the page closes saying "codespace has stopped" does this happen to anyone else?
